### ISSUE TITLE
Todos precisam comitar

### ISSUE THREAD

**Issue by orlandosaraivajr** [jp8002/PI_FATEC_2024_2-Semestre#6]
Na revisão em aula, observei que nem todos os integrantes estão enviando os entregáveis.
Todos estão trabalhando ?

**Comment by DaviBonelli** [comment]
Sim, todos estão colaborando. Nas próximas semanas vamos atualizar o GitHub com todos os entregáveis.
